src/script/auth/page/CustomEnvironmentRedirect.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {ContainerXS, FlexBox, COLOR, InfoIcon, Text, Bold} from '@wireapp/react-ui-kit';
+import {SVGIcon} from '@wireapp/react-ui-kit/dist/Icon/SVGIcon';
+import {UrlUtil} from '@wireapp/commons';
+
+import React, {useState, useEffect} from 'react';
+import {useIntl} from 'react-intl';
+import {connect} from 'react-redux';
+import {AnyAction, Dispatch} from 'redux';
+import {bindActionCreators} from '../module/reducer';
+import Page from './Page';
+import {QUERY_KEY} from '../route';
+import {actionRoot} from '../module/action';
+import {isDesktopApp} from '../Runtime';
+import SVGProvider from '../util/SVGProvider';
+import {customEnvRedirectStrings} from '../../strings';
+
+interface Props extends React.HTMLProps<HTMLDivElement> {}
```

Comment:
only doNavigate, doSendNavigationEvent are used from props, so do we still need this?